decentraland/unity-explorer
# fixed mime header for video streams

## What does this PR change?

Fix #2524
This PR is once again a workaround due to Mac OS limitations.
The problem is that video files coming from our content server are flagged as application/octet-stream, but mac OS without a specific content type cannot play them.
This adds a query param for video files from content server to force the correct content type

## How to test the changes?



1. Launch the explorer
2. Go around in scenes with a video stream like genesis plaza bar or dollhouse
3. Check that the stream is working correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
